README.md

```diff
@@ -45,6 +45,24 @@ $ cekit build --overrides "{version: <version_default_is_latest>}" docker
 
 ### Running the Infinispan Operator
 
+#### Adding Annotations
+
+Annotations are labels that {ispn_operator} propagates to pods and services that it controls.
+You should define custom annotations before you install the Infinispan Operator with the following environment variables:
```

Comment:
Operator Deployment as well as desired variables can be changed at any time. This will occurs operator redeployment and it will start to work with the new parameters